pep-0701.rst

```diff
@@ -320,7 +340,75 @@ All restrictions mentioned in the PEP are lifted from f-string literals, as expl
   expanded when the innermost string is evaluated.
 * Comments, using the ``#`` character, are possible only in multi-line f-string literals,
   since comments are terminated by the end of the line (which makes closing a
-  single-line f-string literal impossible)
+  single-line f-string literal impossible). Comments in multi-line f-string literals require
+  the closing ``{`` of the expression part to be present in a different line as the one the
+  comment is in.
+
+.. _considerations_of_quote_reuse:
```

Comment:
```suggestion
.. _701-considerations-of-quote-reuse:
```

* Namespace ref target label to avoid collisions between PEPs
* Use conventional `-` instead of `_`